integrations/tensorflow/e2e/slim_vision_models/BUILD

```diff
@@ -72,56 +72,36 @@ iree_slim_vision_test_suite(
             ],
         },
         {
-            # Failing all but tf and vmla:
+            # Failing llvmjit and vulkan:
             "models": [
-                "inception_resnet_v2",
-                # tflite: RuntimeError: tensorflow/lite/kernels/reshape.cc:66 num_input_elements != num_output_elements (38400 != -1571481807)Node number 333 (RESHAPE) failed to prepare.
-                # llvmjit: *** Received signal 6 ***
-                # vulkan: Floating point difference was too large. Max abs diff: 1.3961234, atol: 5e-05, max relative diff: 0.27304956, rtol: 1e-06
+                "nasnet_mobile",
+                "nasnet_large",
+                "pnasnet_large",
+                "resnet_v2_50",
                 "resnet_v2_101",
-                # tflite: RuntimeError: tensorflow/lite/core/subgraph.cc BytesRequired number of elements overflowed.
-                # llvmjit: Floating point difference was too large. Max abs diff: 11.668068, atol: 5e-05, max relative diff: 0.93950737, rtol: 1e-06
-                # vulkan: Floating point difference was too large. Max abs diff: 11.668067, atol: 5e-05, max relative diff: 0.93950737, rtol: 1e-06
                 "resnet_v2_152",
-                # tflite: RuntimeError: tensorflow/lite/core/subgraph.cc BytesRequired number of elements overflowed.
-                # llvmjit: Floating point difference was too large. Max abs diff: 7.080696, atol: 5e-05, max relative diff: 0.97750616, rtol: 1e-06
-                # vulkan: Floating point difference was too large. Max abs diff: 7.08069, atol: 5e-05, max relative diff: 0.97750485, rtol: 1e-06
             ],
             "backends": [
-                "tflite",
                 "iree_llvmjit",
                 "iree_vulkan",
             ],
         },
         {
-            # Failing llvmjit and vulkan:
+            # Failing vulkan:
             "models": [
+                # [ERROR]: cannot separate Linalg/Parallel ops into multiple kernels
```

Comment:
Is this tracked somewhere? @MaheshRavishankar ?

Reply:
These tests aren't tracked yet – I'm going to look into adding them to the internal runner, since they don't seem flakey (despite using external data).

Reply:
Oh this seems like a big. Can you tell me how to repro this?